### PR TITLE
Schema update and Documentation

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: balance/handlers/customer_handler.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    schema = {
+        "type": "object",
+        "properties": {
+            "email": {"type": "string"},
+            "name": {"type": "string"},
+            "wallet_address": {"type": "string"},
+            "comment": {"type": "string"},
+        },
+        "required": ["email", "wallet_address", "comment"],
```
Comment (raamb) [inline balance/handlers/customer_handler.py:19]: Lets add a validation that email is not bank and is a valid email format

File: balance/services/balance_service.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        transfer_details = snapshot.find_transfer_status(address)
+
+        if transfer_details is not None:
+            data["transfer_info"] = {
+                "converted_tokens": transfer_details.transfer_amount_in_cogs,
+                "transfer_time": str(transfer_details.transfer_time),
+                "transaction_hash": transfer_details.transfer_transaction,
+            }
+
+        message = "Success"
```
Comment (raamb) [inline balance/services/balance_service.py:26]: Use constants for Success and status code

File: balance/services/balance_service.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        statusCode = 200
+    else:
+        data = None
+        statusCode = 404
```
Comment (raamb) [inline balance/services/balance_service.py:30]: This is not a 404

File: balance/services/customer_service.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from infrastructure.repository.customer_respository import Customer
+
+customer = Customer()
+
+
+def submit_user_question(address, email, comment, name=None):
+    customer.submit_comment(address, comment, email, name)
+    statusCode = 200
```
Comment (raamb) [inline balance/services/customer_service.py:8]: Use constants

Comment (rajeevbbqq) [reply balance/services/customer_service.py:8]: Updated to contstant in this PR #3

File: balance/services/balance_service.py (hunk 3 above)
Comment (rajeevbbqq) [reply balance/services/balance_service.py:30]: Updated in the PR #3

File: balance/services/balance_service.py (hunk 2 above)
Comment (rajeevbbqq) [reply balance/services/balance_service.py:26]: Updated to constant in this PR #3

**Comment by rajeevbbqq** [comment]
I lost committing to the same PR after making the repo public, raised a new PR with code changes here: https://github.com/singnet/token-balances-service/pull/2

**Pull request closed without merging**